Sources/StreamChatSwiftUI/ChatChannel/MessageList/AsyncVoiceMessages/VoiceRecordingContainerView.swift

```diff
@@ -40,32 +40,38 @@ public struct VoiceRecordingContainerView<Factory: ViewFactory>: View {
     
     public var body: some View {
         VStack {
-            if let quotedMessage = utils.messageCachingUtils.quotedMessage(for: message) {
-                factory.makeQuotedMessageView(
-                    quotedMessage: quotedMessage,
-                    fillAvailableSpace: !message.attachmentCounts.isEmpty,
-                    isInComposer: false,
-                    scrolledId: $scrolledId
-                )
-            }
-            
-            ForEach(message.voiceRecordingAttachments, id: \.self) { attachment in
-                VoiceRecordingView(
-                    handler: handler,
-                    addedVoiceRecording: AddedVoiceRecording(
-                        url: attachment.payload.voiceRecordingURL,
-                        duration: attachment.payload.duration ?? 0,
-                        waveform: attachment.payload.waveformData ?? []
-                    ),
-                    index: index(for: attachment)
-                )
+            VStack {
+                if let quotedMessage = utils.messageCachingUtils.quotedMessage(for: message) {
+                    factory.makeQuotedMessageView(
+                        quotedMessage: quotedMessage,
+                        fillAvailableSpace: !message.attachmentCounts.isEmpty,
+                        isInComposer: false,
+                        scrolledId: $scrolledId
+                    )
+                }
+                
+                ForEach(message.voiceRecordingAttachments, id: \.self) { attachment in
+                    VoiceRecordingView(
+                        handler: handler,
+                        addedVoiceRecording: AddedVoiceRecording(
+                            url: attachment.payload.voiceRecordingURL,
+                            duration: attachment.payload.duration ?? 0,
+                            waveform: attachment.payload.waveformData ?? []
+                        ),
+                        index: index(for: attachment)
+                    )
+                }
             }
+            .padding(.all, 8)
+            .background(Color(colors.background8))
+            .cornerRadius(16)
             if !message.text.isEmpty {
                 AttachmentTextView(message: message)
                     .frame(maxWidth: .infinity)
                     .cornerRadius(16)
             }
```

Comment:
This corner radius is not need anymore:
```suggestion
            if !message.text.isEmpty {
                AttachmentTextView(message: message)
                    .frame(maxWidth: .infinity)
            }
```